workers/unity/Packages/com.improbable.gdk.playerlifecycle/Config/PlayerLifecycleConfig.cs

```diff
@@ -2,6 +2,22 @@
 
 namespace Improbable.Gdk.PlayerLifecycle
 {
+    /// <summary>
+    ///     Signature of the delegate called to get a Player Entity Template.
```

Comment:
I'm not a fan of calling this the `Signature of the delegate`. As the delegate _defines_ the signature. 

```
Signature of the function called to get a player <see cref="EntityTemplate"/> instance.
```